README.md

```diff
@@ -8,25 +8,29 @@
 
 This plugin adds Go language support for Vim, with the following main features:
 
-* Build with `:GoBuild`, install with `:GoInstall` or test
-  with `:GoTest` (run single tests via `:GoTestFunc`)
-* Show test coverage with `:GoCoverage` or in browser with `:GoCoverageBrowser` 
-* Goto definition with `:GoDef`
-* Quick jump to declarations with `:GoDecls` or `:GoDeclsDir`
-* Show documentation with `:GoDoc` inside or in browser with `:GoDocBrowser`
-* Quickly execute your current file/files with `:GoRun`
-* Advanced source analysis tools utilizing guru, such as `:GoImplements`,
-  `:GoCallees`, and `:GoReferrers`
-* Change or display `GOPATH` with `:GoPath`
-* Multiple 3rd linter support with `:GoMetaLinter`
-* Renaming identifiers with `:GoRename`
-* Share your code to [play.golang.org](http://play.golang.org) with `:GoPlay`
-* Switch between `*.go` and `*_test.go` code with `:GoAlternate`
-* Add/Remove tags on struct fields with `:GoAddTags`
-* Add import paths via `:GoImport` or remove them with `:GoDrop`
-* Custom vim text objects such as `a function (af)` or `inner function (if)`
-* ... and many more! Please see [doc/vim-go.txt](doc/vim-go.txt) for more information.
-
+* Improved syntax highlighting and folding.
+* Completion support via `gocode`.
+* `gofmt` or `goimports` on save keeps the cursor position and undo history.
+* Go to symbol/declaration with `:GoDef`.
+* Look up documentation with `:GoDoc` or `:GoDocBrowser`.
+* Easily import packages via `:GoImport`, remove them via `:GoDrop`.
```

Comment:
Can you move these top items (from `Improved syntax...` until `Easily import ...`) under the item `Quickly execute your current ...`. The list should start with the `Compile your package... `sentence